docs/maintain/maintain-guides-how-to-nominate-polkadot.md

```diff
@@ -57,25 +58,73 @@ detailed on the [account generation][] page.
+receive rewards, you need at least a balance greater than the minimum active bond. Depending on your
+validators, if your active validator is oversubscribed, you will earn rewards only if your stake is
+within that of the top {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators. If the validator misbehaves, It is worth noting that your
+stake is subject to slashing, irrespective of whether you are in the top {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators or not.
+
+As the minimum active bond is a dynamic value, it can make your nomination inactive when the
+threshold goes above your bonded balance. Hence, to be eligible to earn rewards while nominating,
+you would need to stake a much higher balance than the minimum active bond.
+
+Nomination pools are a way to participate in staking with as little as 1 DOT and earn staking
+rewards. Nomination pools differ from custodial solutions (like staking through central exchanges)
+because they are non-custodial, native to Polkadot's protocol, permissionless, transparent, and run
+in a decentralized way by the community. Before joining a nomination pool, you must ensure that the
+pool is earning rewards and nominating the validators that match your preferences. Participating in
+pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is
+the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you
+are trusting the pool operator to act in your best interests. However, it is advised to check the
+validators nominated by the pool from time to time and change the pool if necessary.
+
+|                                         Nominating                                          |                                                     Joining a Pool                                                      |
+| :-----------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------: |
+|                                 Minimum 10 DOT to nominate.                                 |                                              Minimum 1 DOT to be a member.                                              |
+| Annual return based on past performance is approximately 15% of the bonded funds. | Annual return based on past performance is approximately 15% of the bonded funds. |
```

Comment:
It is not recommended to put a number on the annual return for a nominator or a pool in this table, as it depends on many variables. Some nominators receive more than 20% returns and some around 10%. It is okay to put numbers like 15% inflation as it is coded in the protocol, but not the staking returns. I would suggest removing this. Just to make sure I'm not being too cautious here, tagging @laboon 

```suggestion
| Annual return based on past performance is approximately 15% of the bonded funds. | Annual return based on past performance is approximately 15% of the bonded funds. |
```

Reply:
I'm aware we show this figure on Staking Dashboard. As this is a comparison table between nominating directly vs joining pools, if there is no difference in rewards earned whether nominating directly or through pools, we may have to just highlight that instead.

Reply:
Yeah, I wouldn't include anything about historical returns. It varies too much and isn't really relevant. People who want to know this information can calculate it themselves elsewhere.

You should also continue using the term "rewards", and do not change it to "return". 

Just a note, inflation is targeted at 10%, not 15%.

Reply:
Tab row removed